ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -11,17 +11,19 @@
 
 import org.openmrs.uitestframework.page.Page;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 
 public class AdministrationPage extends Page {
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-mangeUsers a");
+    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
```

Comment:
please move this to its own corresponding ticket and attach commits to legacy repo to it.

Reply:
ok thanks let me create its own separate ticket